.env.example

```diff
@@ -4,4 +4,4 @@ GOOGLE_CLIENT_SECRET=""
 
 # next-auth configuration
 NEXTAUTH_URL="http://localhost:3000"
-NEXTAUTH_SECRET=""
+NEXTAUTH_SECRET=""
```

Comment:
I think you removed the newline at the end here. There shouldn't be a diff here.

Reply:
I don't think this is resolved? At least I don't see changes on my phone